src/test/java/hygieia/transformer/JunitXmlToTestCapabilityTransformerTest.java

```diff
@@ -0,0 +1,58 @@
+
+package hygieia.transformer;
+
+import com.capitalone.dashboard.model.TestCapability;
+import com.capitalone.dashboard.model.TestSuite;
+import com.capitalone.dashboard.model.quality.JunitXmlReport;
+import org.junit.Test;
+
+import javax.xml.bind.JAXBContext;
+import javax.xml.bind.JAXBException;
+import javax.xml.bind.Unmarshaller;
+
+import java.io.File;
+import java.io.FileReader;
+import java.net.URL;
+
+import static org.hamcrest.Matchers.equalTo;
+import static org.hamcrest.Matchers.is;
+import static org.junit.Assert.assertThat;
+
+public class JunitXmlToTestCapabilityTransformerTest {
+
+    JunitXmlToTestCapabilityTransformer sut = new JunitXmlToTestCapabilityTransformer();
+
+    private static  JAXBContext jaxbContext;
```

Comment:
please mark this as `private static final JAXBContext jaxbContext;` as the value is initialized in static block.